101-webapp-including-insights/azuredeploy.json

```diff
@@ -0,0 +1,142 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "webappName": {
+      "type": "string",
+      "defaultValue": "GEN-UNIQUE",
+      "minLength": 1,
+      "metadata": {
+        "description": "The name of the web application"
+      }
+    },
+    "appInsightsName": {
+      "type": "string",
+      "minLength": 1,
+      "defaultValue": "GEN-UNIQUE",
+      "metadata": {
+        "description": "The name of the application insights instance"
+      }
+    },
+    "location": {
+      "type": "string",
+      "minLength": 1,
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "The location for the resources"
+      }
+    },
+   "hostingPlanName": { 
+      "type": "string",
+      "minLength": 1,
+      "defaultValue": "GEN-UNIQUE",
+      "metadata": {
+        "description": "The name of the hosting plan for the web application"
+      }
+    },
+    "skuName": {
+      "type": "string",
+      "defaultValue": "F1",
+      "metadata": {
+        "description": "The hosting plan's pricing tier and capacity. Check details at https://azure.microsoft.com/en-us/pricing/details/app-service/"
+      }
+    },
+    "owner": {
+      "type": "string",
+      "defaultValue": "GEN-UNIQUE",
```

Comment:
this doesn't need to be unique so you can use a literal value here

Reply:
done